### PR TITLE
🚑 Fix h1 tag in skills section

### PR DESCRIPTION
**Pull request by th3c0d3br34ker**

### CODE REVIEW

**Comment by th3c0d3br34ker** [comment]
Fixes #14

File: src/containers/skills/Skills.css [hunk 1]
```diff
@@ -30,7 +30,7 @@
 }
 
 .skills-heading {
-  font-size: 56px;
+  font-size: 52px;
```
Comment (ashutosh1919) [inline src/containers/skills/Skills.css:33]: I think the problem is with `line-height` and not `font-size`

Comment (th3c0d3br34ker) [reply src/containers/skills/Skills.css:33]: Okay then.

**Comment by ashutosh1919** [comment]
@th3c0d3br34ker , Please remove `package-lock.json` from the changes and we are good to go with the merge

**Comment by th3c0d3br34ker** [comment]
Why is `name` different in `package.json` and `package-lock.json` ??

**Comment by ashutosh1919** [comment]
Yeah will have to change that but not in this PR.

**Comment by ashutosh1919** [comment]
@all-contributors please add @th3c0d3br34ker for question

**Pull request merged**